.drone.yml

```diff
@@ -121,6 +121,7 @@ steps:
     commands:
       - docker login ghcr.io --username "$${GHCR_USERNAME}" --password "$${GHCR_PASSWORD}"
       - make PUSH=true
+      - make nonfree PUSH=true
```

Comment:
might be better to move to default list?

Reply:
I would keep them separate if we can... I think that makes it easier (mentally) to keep the distinction between the two